qa/tasks/mgr/mgr_test_case.py

```diff
@@ -105,6 +105,8 @@ def setUpClass(cls):
                 "Only have {0} manager daemons, {1} are required".format(
                     len(cls.mgr_cluster.mgr_ids), cls.MGRS_REQUIRED))
 
+        cls.mgr_cluster.mon_manager.raw_cluster_cmd('config', 'set', 'mds',
+                                                    'defer_client_eviction_on_laggy_osds', 'false')
```

Comment:
having a comment explaining the reason to turn it off would be good

Reply:
Sure